pinot-query-planner/src/main/java/org/apache/calcite/sql2rel/PinotConvertletTable.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.calcite.sql2rel;
+
+import org.apache.calcite.rex.RexBuilder;
+import org.apache.calcite.rex.RexNode;
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+
+/**
+ * PinotConvertletTable is a wrapper of {@link StandardConvertletTable} with the customizations of not converting
+ * certain SqlCalls, e.g. TIMESTAMPADD, TIMESTAMPDIFF.
+ */
+public class PinotConvertletTable implements SqlRexConvertletTable {
+
+  public static final PinotConvertletTable INSTANCE = new PinotConvertletTable();
+
+  private PinotConvertletTable() {
+  }
+
+  @Override
+  public @Nullable SqlRexConvertlet get(SqlCall call) {
+    switch (call.getKind()) {
+      case TIMESTAMP_ADD:
+        return TimestampAddConvertlet.INSTANCE;
+      case TIMESTAMP_DIFF:
+        return TimestampDiffConvertlet.INSTANCE;
```

Comment:
Does it work if we simply do
```suggestion
      case TIMESTAMP_ADD:
      case TIMESTAMP_DIFF:
        return null;
```

Reply:
Then there is no way to convert SqlNode to RexNode.
There has to be an implementation for every sqlkind.